graylog2-web-interface/src/components/errors/ReportedError.jsx

```diff
@@ -4,10 +4,11 @@ import { useState, useEffect } from 'react';
 import { withRouter } from 'react-router';
 
 import ErrorsActions from 'actions/errors/ErrorsActions';
-import { type ReportedError as ReportedErrorType, ReactErrorType, UnauthorizedErrorType } from 'logic/errors/ReportedErrors';
+import { type ReportedError as ReportedErrorType, ReactErrorType, UnauthorizedErrorType, StreamPermissionErrorType } from 'logic/errors/ReportedErrors';
 
 import RuntimeErrorPage from 'pages/RuntimeErrorPage';
 import UnauthorizedErrorPage from 'pages/UnauthorizedErrorPage';
+import StreamPermissionErrorPage from 'pages/StreamPermissionErrorPage';
 
 const ReportedError = ({ children, router }) => {
```

Comment:
This component needs a better name, imo. Maybe something like `ReportedErrorBoundary`? It is not a react error boundary, nevertheless that's what it is doing: Rendering its children _or_ an error if present. Renaming the component also removes the need to rename the `ReportedError` type import.

Reply:
I agree, my only problem was I connect "ErrorBoundary" in React with `componentDidCatch`, but of course it is a more meaningful name.